src/finitio/support/constraint.coffee

```diff
@@ -13,7 +13,7 @@ class Constraint
   isAnonymous: ->
     !@name?
 
-  accept: (arg) ->
+  accept: (arg, world) ->
     throw new Error("Constraint is an abstract class")
```

Comment:
On second thoughts, I don't agree with this, but making something better requires some refactoring. Let keep it for now.